test/e2e/tests/send-edit.spec.js

```diff
@@ -68,10 +68,10 @@ describe('Editing Confirm Transaction', function () {
           '.currency-display-component__text',
         );
         const editedTransactionAmount = editedTransactionAmounts[0];
-        assert.equal(await editedTransactionAmount.getText(), '2.2');
+        assert.equal(await editedTransactionAmount.getText(), '0.0008');
```

Comment:
Why is this e2e test change needed?

Reply:
Because we switched the order of gas fee and total with the new design.